bin/desi_dailyproc

```diff
@@ -54,6 +54,13 @@ def parse(options=None):
                         help="Number of submissions to make to cori at a time."+
                              "After 'nsubmits,' the script will wait 'pausetime'"+
                              "minutes before submitting another 'nsubmits.'")
+
+    parser.add_argument("--cameras", type=str, required=False, default='None',
+                        help="Explicitly define the spectrographs for which you want"+
+                             " to reduce the data. Should be a comma separated list."+
+                             " Numbers only assumes you want to reduce R, B, and Z "+
+                             "for that camera. Otherwise specify separately [BRZ|brz][0-9].")
```

Comment:
See comment below about default `None` vs. `"None"` and camera parsing.  I like the idea of allowing a single number to mean b,r,z for that spectrograph (thanks for documenting!).  If that is more flexible parsing than what `desi_proc` supports, let's move that into `desi_proc` itself so that the `--cameras` parsing is only done in one place and is consistent, and `desi_dailyproc` is just passing forward whatever it gets to `desi_proc`.